code/src/index.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import './index.css'
-import { App } from './App'
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import { App } from './App';
+import './index.css';
```

Comment:
Duplicate import. PS. There is a convention that says that we should divide external imports from local ones, and the styling should come last:

```
import React from 'react';
import ReactDOM from 'react-dom';

import { App } from './App';

import './style.css';
```